src/portfft/specialization_constant.hpp

```diff
@@ -44,6 +44,12 @@ constexpr static sycl::specialization_id<level> GlobalSubImplSpecConst{};
 constexpr static sycl::specialization_id<Idx> GlobalSpecConstLevelNum{};
 constexpr static sycl::specialization_id<Idx> GlobalSpecConstNumFactors{};
 
+constexpr static sycl::specialization_id<bool> SpecConstTakeConjugateOnLoad{};
+constexpr static sycl::specialization_id<bool> SpecConstTakeConjugateOnStore{};
```

Comment:
It is more verbose but these should use a new enum similarly to `elementwise_multiply` and `apply_scale_factor`

Reply:
my initial approach was to create yet another enum,
but I switched to using bool at the end. It's just that I think that way we'll end up creating quite a few enums with very little utility, where we only need an `on/off` switch.

Reply:
Yes I am wondering about that. In the current usages that you have I think it is ok to have them as bool. I think in the long term we wanted to avoid functions with lots of boolean which can be confusing on the call sites.

I don't really like the inconsistency but if no one else is bothered enough I'll just resolve this comment.

Reply:
I dislike the inconsistency

Reply:
I have changed it to an enum for now.
>I think in the long term we wanted to avoid functions with lots of boolean which can be confusing on the call sites
That was when these were still template parameter. Now that they are well named spec_constants, readability will not be hampered either way.